### ISSUE TITLE
Remove the requirement of the featured article to have an image

### ISSUE THREAD

**Issue by amywtlin** [forem/forem#15292]

**Comment by amywtlin** [comment]
@michael-tharrington I created an issue for removing the requirement of having to have a cover image to feature a post. Please help check it looks fine!

@jeremyf Just to double confirm, as this is part of the feed experiment, it means it would only roll out to a selection of DEV audience first. How does that affect Creators? Will they have to wait for 100% rollout to benefit from this change?

**Comment by michael-tharrington** [comment]
Nice! I think this definitely makes sense, particularly for the "latest" feed.

**Comment by jeremyf** [comment]
@amywtlin (and cc @michael-tharrington)  is there a case where we want admin's to configure this?

That is to say a site creator could require that featured posts must have images.  If so, that impacts implementation in that we'd want to add an admin setting.

**Comment by jeremyf** [comment]
We'll need to assess the impact of not requiring a main image.  Here are some places to consider in the codebase.

- [ ] https://github.com/forem/forem/blob/eec738710b4963e595c3128ec49cb276e675e6f2/app/javascript/articles/Feed.jsx#L34-L40
- [ ]  https://github.com/forem/forem/blob/eec738710b4963e595c3128ec49cb276e675e6f2/app/javascript/articles/Article.jsx#L41-L43
- [ ] https://github.com/forem/forem/blob/eec738710b4963e595c3128ec49cb276e675e6f2/app/views/articles/_single_story.html.erb#L6-L12

**Comment by amywtlin** [comment]
>is there a case where we want admin's to configure this?

I don't think so. If that is the desired experience, there are a few easy workarounds I can think of:
1. admins can just go ahead and add an image to the post they are going to feature. 
2. admins can just pick posts with images to feature

@michael-tharrington Is that correct, though?

**Comment by jeremyf** [comment]
@amywtlin I think we need to revisit this and re-scope given our current understanding.

**Comment by michael-tharrington** [comment]
Shoot! My apologies for missing [your question above](https://github.com/forem/forem/issues/15292#issuecomment-964736505) @amywtlin!

I could see it being cool to give admin the choice of whether they want that top post to always have a cover image or not. But, I don't feel strongly about it!

I think usually admins would refrain from adding a cover image to a user's post. I know our protocol on DEV and CN is not to edit a user's post unless we've asked them for explicit permission to do so and told them what our plans are.

The top post is generally chosen automatically unless an admin chooses to pin a post, in which case, you're right that they could choose to avoid posts without cover images if that's their preference.

As for revisiting this at a later date, that's fine by me. This does feel like a fairly low priority thing to me, but it did come from a genuine interaction with Forem admin and not just from the gut, haha.

**Comment by msarit** [comment]
@amywtlin What is the verdict on the implementation of this ticket? 🙏🏾

**Comment by amywtlin** [comment]
@msarit I had a discussion with Jeremy on this and we agree this should be thought through thoroughly before we touch this. I am moving this back to `Ideas/RFCs`.
